### PR TITLE
Product model#2

### PR DESCRIPTION
**Pull request by luisamiranda**
Preliminary model created

### CODE REVIEW

File: db/models/product.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Product = db.define('products', {
+    title: {
+        type: Sequelize.STRING,
+        isUnique: true,
```
Comment (illuminatedspace) [inline db/models/product.js]: isUnique is not a validator. I think we need to do this in the route.

File: db/models/product.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        type: Sequelize.INTEGER,
+        allowNull: false,
+    },
+    inventory: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+    },
+    photo: {
+        type: Sequelize.STRING,
+        defaultValue: 'https://pixabay.com/photo-1752164/',
```
Comment (illuminatedspace) [inline db/models/product.js:25]: Terrifying stock photo. Ace choice.

**Review by illuminatedspace — CHANGES_REQUESTED** [review]
I think there are some changes to make.

File: db/models/product.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    category: {
+        type: Sequelize.STRING,
+        notEmpty: true,
+    },
+}, {});
+
+module.exports = Product
```
Comment (illuminatedspace) [inline db/models/product.js:33]: I think that you might want to set up your associations in the index folder.

File: db/models/product.js (hunk 1 above)
Comment (luisamiranda) [reply db/models/product.js]: excellent point. removed.

**Review by illuminatedspace — APPROVED** [review]
Approved!

**Pull request merged**